campgns/ami2019/MAP10001.txt

```diff
@@ -129,9 +134,12 @@ REM	QUICK_OBJECTIVE(3,"Locate and deploy any traps you can find to prevent these
 ENDIF
 
 IF(PLAYER0,RESEARCH>0)
-REM	QUICK_OBJECTIVE(4,"As the library becomes a part of your dungeon, you have a strange vision uncovering a small area. Getting to the spell as soon as possible should be your top priority. Acquiring 32000 pieces of gold is our goal.",5)
-	DISPLAY_OBJECTIVE(6,PLAYER0)
-	REVEAL_MAP_LOCATION(PLAYER0,5,18)
+	REM only display the message if player has not visited it yet
+	IF(PLAYER0,FLAG0==0)
+	REM	QUICK_OBJECTIVE(4,"As the library becomes a part of your dungeon, you have a strange vision uncovering a small area. Getting to the spell as soon as possible should be your top priority. Acquiring 32000 pieces of gold is our goal.",5)
+		DISPLAY_OBJECTIVE(6,PLAYER0)
```

Comment:
I see the objective eye used to be '5' and is 'PLAYER0' now. Did we break all your objective eyes?